package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-webrtc",
-  "version": "1.75.0-hive-1",
+  "version": "1.76.0-hive-1",
```

Comment:
The second number in versioning is used for WebRTC version, both by [upstream repository](https://github.com/react-native-webrtc/react-native-webrtc) and our fork.
In this particular case we are not updating WebRTC, just adding an extra fix on top of M75. More specifically, we are adding our fix on top of `1.75.0` version of upstream. So the correct version number would be `1.75.0-hive-2`.

Reply:
Just updated, thanks for the comment.